### PR TITLE
fix[N12] Fix typographical errors

### PR DESCRIPTION
**Pull request by chrismaree**
In HubPool.sol:
line 99: "Heler" should be "Helper"
line 201: "proposal" should be "Proposal"
line 235: "its" should be "it's"
line 294: "disputes.." should be "disputes."
line 377: "for again" should be "again."
line 419: "access more funds that" should be "to access more funds than"
line 475: "to along" should be "along"
line 480: "leafs" should be "leaves"
line 532: "neccessary" should be "necessary"
line 568: "to back" should be "back"
line 569: "leafs" should be "leaves"
line 569: "wont" should be "won't"
line 865: "timeFromLastInteraction ,undistributedLpFees)" should be "timeFromLastInteraction,
undistributedLpFees)"
line 866: "a fees." should be "fees."
line 913: "decrease" should be "decreased"
line 962: "send" should be "sent"
In HubPoolInterface.sol:
line 13: "sent" should be "send"
In MerkleLib.sol:
line 86: "\*" should be "*"
In Polygon_SpokePool.sol:
line 43: "priviledges" should be "privileges"
In SpokePool.sol:
line 55: "token" should be "chain"
line 67: "leafs" should be "leaves"
line 292: "users" should be "user's"
line 347: "receipient." should be "recipient."
In SpokePoolInterface.sol:
line 11: "inverted." should be "negated."
line 27: "a the" should be "the"

### CODE REVIEW

**Comment by chrismaree** [comment]
Another change I made was to replace all instances of `leafs` with `leaves` through the repo.

**Review by mrice32 — APPROVED** [review]
Missed:
- HubPool: line 294: "disputes.." should be "disputes."

File: contracts/SpokePoolInterface.sol [hunk 1]
```diff
@@ -8,7 +8,7 @@ interface SpokePoolInterface {
     // This leaf is meant to be decoded in the SpokePool to pay out successful relayers.
     struct RelayerRefundLeaf {
         // This is the amount to return to the HubPool. This occurs when there is a PoolRebalanceLeaf netSendAmount that is
-        // negative. This is just that value inverted.
+        // negative. This is just the negative of this value.
```
Comment (nicholaspai) [inline contracts/SpokePoolInterface.sol:11]: Make sure this change is consistent with #102 !

**Review by nicholaspai — APPROVED** [review]
LGTM! Just one tiny nit

**Pull request merged**